cirq-core/cirq/contrib/svg/svg.py

```diff
@@ -11,6 +11,7 @@
 
 def fixup_text(text: str):
     if '\n' in text:
+        # TODO: Custom MatrixGate
```

Comment:
nit: link to issues of outstanding svg issues so we can track this todo

Reply:
Great! Added :heavy_check_mark: